.gitignore

```
@@ -3,6 +3,7 @@
 *.jl.mem
 .DS_Store
 /Manifest.toml
+test/Manifest.toml
```

This will ignore any file of name `Manifest.toml` that is in a subfolder `test` (not necessarily a subfolder of the root directory). If you want to ignore specifically the one in the test folder, you should write
```suggestion
/test/Manifest.toml
```
Alternatively, you could just change the line above to
```suggestion
Manifest.toml
```
to ignore all Manifest.toml files in any subfolder.